.gitignore

```diff
@@ -7,5 +7,6 @@
 *.iws
 *.xml
 *.class
```

Comment:
It is redundant as all compile classes would be in the target directory and you have already added this directory into gitignore.
*.xml - you will ignore pom.xml. But it should be so.